### PR TITLE
check if enough credits for each post before showing player. added fi…

### PR DESCRIPTION
**Pull request by borisjaskerovich**
…xing to some bugs.

### CODE REVIEW

**Comment by shlomisut** [comment]
the credits check should not be in the plugin code. it should be done during player load on "sas" side

File: public/class-amazonpolly-public.php [hunk 1]
```diff
@@ -103,76 +103,78 @@ public function content_filter( $content ) {
...
-							}
-						}
-					}
-
-					$subscribe_part = $this->get_subscribe_part();
-
-
-					$polly_content = '
+        // Check if Amazon Polly is enabled in WP, if yes then...
+        if(($common->is_trinity_connected() && $common->is_trinity_enough_credits($post_id)) || !$common->is_trinity_connected()){
```
Comment (shlomisut) [inline public/class-amazonpolly-public.php]: this is an exit condition - you added it and had to indent a huge pile of code. 
can you make it nicer ?

Comment (borisjaskerovich) [reply public/class-amazonpolly-public.php]: it is not an exit condition. if the condition is true then the code continues to execute.

Comment (shlomisut) [reply public/class-amazonpolly-public.php]: doesn't matter if it's true or false, the facts that it's all or nothing makes it an exit condition

Comment (borisjaskerovich) [reply public/class-amazonpolly-public.php]: i don't how to change it. there is still code after the condition that should run any way.

Comment (shlomisut) [reply public/class-amazonpolly-public.php]: extract the condition to a method, and add it to the lower one `if ($common->is_polly_enabled() && trinity_enabled())`

**Review by shlomisut — APPROVED** [review]

**Pull request merged**